### DISCUSSION TITLE
is it posible to detect image transparency?

### DISCUSSION THREAD

**Discussion by jeetiss** [imgproxy/imgproxy#1218] [Q&A] [answered]
I want to check if a PNG image has transparency. Is it possible with `imgproxy`?

**Comment by DarthSim** [comment]
Not yet, but it's possible to add this to the [info](https://docs.imgproxy.net/getting_the_image_info) endpoint response.

**Reply by jeetiss** [reply]
Thanks for the answer! Do you have any estimation of when this feature will be available?
Can I help with development?

**Reply by DarthSim** [reply, marked as answer]
Sorry, I added this feature and forgot to mention it here 😓 https://docs.imgproxy.net/usage/getting_info#alpha
